apps/src/codebridge/Console/RightButtons.tsx

```diff
@@ -66,7 +71,8 @@ const RightButtons: React.FunctionComponent<RightButtonsProps> = ({
           color={'black'}
         />
       </WithTooltip>
-      {!isShareView && <SwapLayoutDropdown />}
+      {!isShareView && !isWidgetView && <SwapLayoutDropdown />}
+      {isWidgetView && <SettingsButton />}
```

Comment:
In widget view, the code editor whose panel header includes the settings button is hidden. Therefore, we display the settings button so that the user can still access settings in widget view.